content/ui-patterns/feature-onboarding.mdx

```diff
@@ -0,0 +1,271 @@
+
+**In-product marketing empty state**
+
+A first time experience may be more unique than the typical blank state. For special occasions, take a more branded approach to engage and guide the user through complex experiences. Be aware of how the experience will change once the first time UI is no longer there. 
+
+<Flex alignItems="center" flexDirection="column" my={5}>
+  <BorderBox width={12 / 12} border="1px solid" borderColor="gray.3" mb="2">
+    <img src="https://user-images.githubusercontent.com/6744014/88744262-b3cac300-d0fb-11ea-9752-c43722a30399.png" />
+  </BorderBox>
+  <p>GitHub Actions branded first time user experience</p>
+</Flex>
+
+
+# Designated feature discovery areas
+Designated feature area generally detached from where the feature actually lives and have a clear click through to try out the feature or learn more.
+
+## Dashboard feature bulletin
+The top of the right sidebar in a user’s dashabord (logged-in home) is a designated area for feature discovery. They take the user directly to the feature in the product or the relevant marketing pages.
+
+**Basic announcement guidelines**
+• Limit messages to around 160 characters. 
+• If there is a click through, include a clear call-to-action button; avoid hidden links.
+• Use color, with discretion, to spotlight and bring brand feature personality to the annoucement.
+
+<Flex alignItems="center" flexDirection="column" my={5}>
+  <BorderBox width={12 / 12} border="1px solid" borderColor="gray.3" mb="2">
+    <img src="https://user-images.githubusercontent.com/66705495/88829133-09957e80-d181-11ea-9ab2-81a6c5513e20.png" />
+  </BorderBox>
+  <p>GitHub Actions branded first time user experience</p>
+</Flex>
+
+## Beta feature preview dialog
+Feature previews allow teams to test features in production with users who opt in. A feature under the beta preview will be added to the Feature Preview dialog, accessible through the user dropdown
+
+Please note, features in Beta testing are strongly advised to have a comprehensive release and redaction plan. No feature should remain in Beta for longer than X months unmonitored.
+
+<Flex alignItems="center" flexDirection="column" my={5}>
+  <BorderBox width={12 / 12} border="1px solid" borderColor="gray.3" mb="2">
+    <img src="https://user-images.githubusercontent.com/66705495/88829215-2762e380-d181-11ea-956e-8cf4166ac92c.png" />
+  </BorderBox>
+  <p>GitHub Actions branded first time user experience</p>
+</Flex>
+
+
+# Flairs
+Flairs are subtler ways to accentuate new features. They are a good alternative to call attention to new things without derailing the user from their primary job to be done. 
+
+Consider using flairs for features that appear in a user’s current view but may navigate away from their current page. They may also be applicable for smaller features that appear inline that more self-explainatory and do not need a teaching bubble. 
+
+See Primer documentation on labels and muted links for detailed implementation guidelines.
```

Comment:
Does this exist? 😅

Reply:
[Labels](https://primer.style/css/components/labels) and [muted links](https://primer.style/css/components/labels).